docs/.vuepress/components/FaqItem.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <article class="faq">
+    <h3 :id="slugify(question)">
+      <a :href="`#${slugify(question)}`" aria-hidden="true" class="header-anchor">#</a> {{ question }}
+    </h3>
+
+    <main v-html="content"></main>
+
+    <aside v-if="hasLinks">
+      <useful-links>
+        <useful-links-section v-for="(links, section) in links" :title="section">
+          <ul>
+            <li v-for="link in links">
+              <a :href="link.href">{{ link.text }}</a>
+            </li>
+          </ul>
+        </useful-links-section>
+      </useful-links>
+    </aside>
+
+    <footer class="page-edit">
+      <div class="edit-link">
+        <a :href="githubPath" target="_blank" rel="noopener noreferrer">
+          Edit this FAQ item on Github
+        </a>
+        
+        <OutboundLink />
+      </div>
+    </footer>
+  </article>
+</template>
+
+<script>
+export default {
+  props: {
+    path: {
+      type: String,
+      required: true
+    },
+    question: {
+      type: String,
+      required: true
+    },
+    content: {
+      type: String,
+      required: true
+    },
+    links: {
+      type: Object,
+      default() {
+        return {}
+      }
+    }
+  },
+  computed: {
+    githubPath() {
+      return `https://github.com/dobromir-hristov/vuecommunity/edit/master/${this.path}`
```

Comment:
It would be better to read this from the config.

Reply:
Agreed.

Reply:
I am not sure how to pass the github repo from the plugin config along to the `FAQItem.vue` component, without exposing it on the FAQ's `pageData` (so that the component can access it via `$page`) - thoughts?

Reply:
Also fixed.